.gitignore

```diff
@@ -89,4 +89,4 @@ venv/
 .spyderproject
 
 # Rope project settings
-.ropeproject
\ No newline at end of file
+.ropeproject
```

Comment:
Why doesn't it show the new line I added? 
<img width="298" alt="Screenshot 2022-11-16 at 01 56 22" src="https://user-images.githubusercontent.com/19777158/202023700-0e9cff15-6a4e-405d-a658-d255c3f2918d.png">

Reply:
Got it, thank you for the help.